src/commands/find.ts

```diff
@@ -213,7 +213,7 @@ export default new Command({
 					userDetail.findMember.hoursPerWeek ?? 0,
 					projectNames,
 					sprintf(LINK.USER, frenUser.id),
-					frenUser.id
+					frenUser.username
```

Comment:
will keep using id, because in Embed Msg, mentions are just a tag, and mentioned users will not get notified.

```ts
sprintf(
	"🛠**Skills**: %s\n\n🗓️**Availability**: %f h/week\n\n🌳**Projects**: %s\n\n🔗Click [here](%s) to see <@%s>'s profile",
	topSkills,
	userDetail.findMember.hoursPerWeek ?? 0,
	projectNames,
	sprintf(LINK.USER, frenUser.id),
	frenUser.id
)
```
Also, I put `<@%s>` in the template message, if you are willing to use `frenUser.username`, change `<@%s>` to `%s`